stable/consul-exporter/values.yaml

```diff
@@ -0,0 +1,44 @@
+# Default values for consul-exporter.
+
+replicaCount: 1
+
+restartPolicy: Always
+
+image:
+  repository: prom/consul-exporter
+  tag: master
+  pullPolicy: IfNotPresent
+
+#Consul server host to connect to
+consulServer: localhost
```

Comment:
Unless I'm grossly misreading, I don't think this value will ever work, right? Unless this is running as a sidecar to the actual Consul container?

Reply:
yes, it's sidecar and connect to the actual container to pull in metrics

Reply:
@timstoop is correct. The exporter runs in its own pod, not as a sidecar.

Reply:
In that case I would recommend commenting this out. Does not make sense in any case to have it set to localhost, imho.